src/api/modelTypes.js

```diff
@@ -145,49 +108,155 @@ export type User = $ReadOnly<{|
    */
   avatar_url: AvatarURL,
 
-  // These properties appear in data from the server, but we ignore
-  // them. If we add these, we should try to avoid `avatar_url`
-  // falling out of sync with them.
-  // avatar_source: mixed,
-  // avatar_url_medium: mixed,
-  // avatar_version: mixed,
-
-  // profile_data added in commit 02b845336 (in 1.8.0);
-  // see also e3aed0f7b (in 2.0.0)
-  // (This one doesn't appear in `/users` responses.)
-  profile_data?: empty, // When we need this, describe its actual type.
+  // If we use this, avoid `avatar_url` falling out of sync with it.
+  -avatar_version: number,
```

Comment:
api types: Add write-only `avatar_version` to User type

This doesn't end up having that effect, because the object type is wrapped in `$ReadOnly<…>`.

In order to make it possible to start marking properties as write-only, you can have a prep commit replace the `$ReadOnly` with a `+` on each individual property. For an example: f551158c990c6f169694afa46e69fea33054ced9